Dockerfile

```diff
@@ -24,7 +24,7 @@ WORKDIR /opt/$NAME
 COPY ./app /opt/$NAME/app
 RUN chown $USER:$USER /opt/$NAME
 
-EXPOSE 5000
+EXPOSE 30518
```

Comment:
Please see the related comment on a different PR.

Reply:
this still needs to be reverted

Reply:
thanks, the change wasn't committed yet.